mautic/mautic
# Set Contact's company as primary

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | pending
| Related developer documentation PR URL | na
| Issues addressed (#s or URLs) | n
| BC breaks? | n
| Deprecations? | n

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR gives you the ability to set a primary company for a contact.

#### Steps to test this PR:
1. Apply this PR
2. Go to any contact that has a company assigned, ideally more than one company
3. click on the star icon next to the company name to set it as a primary company, the star color should change to mautic yellow to indicate its a primary company.
